### ISSUE TITLE
Change Intrinsic parameters to fit with own camera

### ISSUE THREAD

**Issue by AIMads** [EryiXie/PlaneRecNet#13]
Hey! 

Is it possible to change intrinsic parameters to fit with your own camera? Or is the model not affected by it?

**Comment by EryiXie** [comment]
Yes, if you want to train the network on a new dataset, you can just reimplement the function to read the camera intrinsic. 
https://github.com/EryiXie/PlaneRecNet/blob/cea1e8b1edf054f59e15891fa799cdcc5feda72d/data/datasets.py#L174

But for inferencing with any data samples, the camera intrinsic is not asked to be given. However, [rumors](https://openaccess.thecvf.com/content_CVPR_2019/html/Facil_CAM-Convs_Camera-Aware_Multi-Scale_Convolutions_for_Single-View_Depth_CVPR_2019_paper.html) said neural networks learn the camera parameters implicitly if trained for monocular depth estimation in the most vanilla ways, and therefore they can not generalize well for other camera models.
